controllers/openshift_controller.go

```diff
@@ -1727,14 +1742,119 @@ func (r *KataConfigOpenShiftReconciler) enablePeerPods() error {
 		},
 	}
 
-	err = r.Client.Create(context.TODO(), &peerpodconf)
+	err := r.Client.Create(context.TODO(), &peerpodconf)
 	if k8serrors.IsAlreadyExists(err) {
```

Comment:
Does this mean that the two MCs and the runtime class are assumed to be created already if the peer pod config is already there ? I'm asking because if a previous call to this function created the peer pod config but not all the other objects it looks like it will never try to create them again.

Reply:
Right, it won't create the objects if peerpodconfig already exists.  Do you think it's too restrictive and better to handle this case?

Reply:
I'd tend to say yes but since this code predates the PR I guess it can be addressed separately.

Reply:
@gkurz I went ahead and handled the case of existing peerpodconfig.